jaxrs/src/main/java/org/killbill/billing/jaxrs/json/CatalogJson.java

```diff
@@ -375,22 +426,204 @@ public int hashCode() {
         }
     }
 
+    public static class TieredBlockJson {
```

Comment:
Can you try to run the [Ruby code generator](https://github.com/killbill/killbill-java-parser) (see the [example](https://github.com/killbill/killbill-client-ruby/blob/master/gen_config/model.conf))?

This generator takes care of sync'ing the JAX-RS models to their Ruby equivalent (https://github.com/killbill/killbill-client-ruby/tree/master/lib/killbill_client/models/gen). We've had sometimes issues in the past where inner classes would confuse the tool. No need to check-in anything / creating a new PR, let's just make sure the Ruby classes are properly generated.

Thanks!